Project Management Process.md

```diff
@@ -0,0 +1,293 @@
+# IPFS Project Management Process
+
+**Status: work in progress, not even a full draft yet**
+
+## Introduction
+TODO: describe the purpose of this document
+
+- this is an opt-in process, nobody in the community is *required* to use this process
+
+## Table on Contents
+- [TL;DR](#tldr)
+- [Projects](#projects)
+- [Roles](#roles)
+  - [Product Owners](#product-owners)
+  - [Project Leads](#project-leads)
+  - [Project Managers](#project-managers)
+- [Structure](#structure)
+  - [Overview](#overview)
+  - [Goals](#goals)
+  - [Backlog](#backlog)
+  - [Milestones](#milestones)
+  - [Project Roadmap](#project-roadmap)
+  - [Organization Roadmap](#organization-roadmap)
+- [Flow](#flow)
+  - [Quarterly Planning](#quarterly-planning)
+  - [Software Development Pipeline](#software-development-pipeline)
+    - [Kanban Pipeline](#kanban-pipeline)
+- [Implementation](#implementation)
+
+## TL;DR
+
+The IPFS project management process has the following **cadence**:
+
+Roadmap Updates
+- every 3 months
+- identify, discuss and plan the next steps for the organization as a whole
+
+"Sprint" Updates
+- once a week (2 weeks?)
+- discuss last week's progress
+- discuss plan for the next week
+
+The IPFS project management process breaks down to the following **units of work**:
+
+Projects 
+- may contain other projects
+- have a project leader who "roadmaps" (decides what are goals and milestones, and when they should happen)
+- have a roadmap (list of milestones and goals)
+- have a backlog (list of goals)
+
+Milestones 
+- measurable unit of progress
+- contains a list of goals, ordered by priority
```

Comment:
I don't think it is a good metaphor, and I think tasks would be better, if we made sure that they are action orientated. I don't think that depending on the lexical semantics of goal having an end-state as opposed to task is a useful distinction, because this really is something that we need to work on, as writers of goals/tasks: instead of relying on there being an end state because of it should be a goal, we should work to make sure that every task is actionable and completable. Put in other words, I don't think the tradeoff will cause us to work less, and I think that having 'task' instead of 'goal' will push us to define our terms better. 

This is something we have not previously done well as a team, and we'll need to grow to do it right.